src/main/scala/cromwell/server/CromwellServer.scala

```diff
@@ -15,7 +15,15 @@ object CromwellServer extends WorkflowManagerSystem {
   val conf = ConfigFactory.load()
   val service = actorSystem.actorOf(CromwellApiServiceActor.props(workflowManagerActor, conf), "cromwell-service")
   val webserviceConf = conf.getConfig("webservice")
-  service.bindOrShutdown(interface = webserviceConf.getString("interface"), port = webserviceConf.getInt("port")) onSuccess {
-    case _ => actorSystem.log.info("Cromwell service started...")
+
+  def run(): Int = {
+    val interface = webserviceConf.getString("interface")
+    val port = webserviceConf.getInt("port")
+    val futureBind = service.bindOrShutdown(interface = interface, port = port)
```

Comment:
Crap. With everything we know now, that's a feature lacking in lenthall. Lenthall shouldn't _always_ be terminating the system.